### ISSUE TITLE
NOT Working 

### ISSUE THREAD

**Issue by GekkoFreak** [NYAN-x-CAT/Lime-Miner#2]
Where is the file where I can build my miner ? I am missing that

**Comment by NYAN-x-CAT** [comment]
You don't need it. Just insert you settings on XMR dialog.

**Comment by NYAN-x-CAT** [comment]
What do you mean?

**Comment by GekkoFreak** [comment]
A .exe file where I can build the stub. How can I build a stub ? 
If I look at your RAT then I need to execute a .exe file and I go to application. How does this work with the miner ?

**Comment by NYAN-x-CAT** [comment]
You go to  ::Build:: and press Build.. that's it

**Comment by GekkoFreak** [comment]
I dont have that file. I disable my AV and i run it on a vm

**Comment by FilterUnfiltered** [comment]
it doesn't work like that, you don't see if it's working or not, you execute it and then wait like 10 minutes and then go to your pool and see if you're getting money or not. I tested this before it was released, or well, I tested the miner on LimeRAT before release which has the same code as this, but anyways, there is nothing wrong with the miner you just don't know how to use it. Look up how to use silent miners and then stop complaining.
